src/App.tsx

```diff
@@ -1,24 +1,234 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  useState,
+  useEffect,
+  useMemo,
+  useCallback,
+} from 'react';
+import {
+  getTodos,
+  deleteTodo,
+  updateTodo,
+  createTodo,
+} from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Error } from './types/Errors';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { FilterBy } from './types/TodosFilter';
+import { TodoFilter } from './components/TodoFilter';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 10353;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(FilterBy.ALL);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleError = useCallback((error: string) => {
+    setErrorMessage(error);
+    setTimeout(() => {
```

Comment:
Do it in the `Error` component
Bc you need make a `clearTimeout` when component was deleted